### PR TITLE
Fix for getobjectacl and listobjectv1 S3 ops

### LINKED ISSUE #3726: Change GetobjectACL response validation from displayname to ID
get_acl_res["Grants"][0]["Grantee"]["DisplayName"] == "NooBaa"
        ), "Invalid grantee"
E       KeyError: 'DisplayName'

### PR DESCRIPTION
**Pull request by parikshithb**
bz: https://bugzilla.redhat.com/show_bug.cgi?id=1917677#c3
Fixes #3726 
Skipping listobjectsv1 on azure resource

Signed-off-by: Parikshith <pbyregow@redhat.com>

### CODE REVIEW

File: tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops.py [hunk 1]
```diff
@@ -226,17 +226,13 @@ def test_mcg_namespace_basic_s3_ops(
...
-            bucketname=ns_bucket,
-            max_keys=max_keys,
-            marker=first_page_res["NextMarker"]
-            if platform == constants.AZURE_PLATFORM
-            else last_key,
-        )
-        get_list_and_verify(
-            next_page_res, obj_keys[mid_index:], "Contents", version="v1"
-        )
+        if not platform == constants.AZURE_PLATFORM:
```
Comment (fbalak) [inline tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops.py:229]: Please add some log message that this check is skipped when Azure platform is used. Something like:
```
else:
    log.warning('Skipping next page entries check for ListObjectV1 - not supported on Azure')
    log.warning('For more info see: https://bugzilla.redhat.com/show_bug.cgi?id=1917677')
```

**Comment by ebenahar** [comment]
@parikshithb thanks for handling this. Shouldn't we maintain a workaround issue so once https://bugzilla.redhat.com/show_bug.cgi?id=1917677 is fixed, we will revert the changes?

**Review by ebenahar — APPROVED** [review]

**Review by fbalak — COMMENTED** [review]
Looks good but I think that there should be some log message when Azure is used that informs about that the check is not happening.

File: tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops.py (hunk 1 above)
Comment (parikshithb) [reply tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops.py:229]: Done. Thank you.

**Comment by parikshithb** [comment]
@ebenahar Since display name varies on different aspects, ID is the best way to validate the Grantee in this case. So this will not be reverted unless there is a major change.  Explained better here: https://bugzilla.redhat.com/show_bug.cgi?id=1917677#c3.

**Review by shreekarSS — APPROVED** [review]
LGTM!

**Review by fbalak — APPROVED** [review]

**Review by Akarsha-rai — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**